core/rfb.js

```diff
@@ -11,7 +11,6 @@
  */
 
 import * as Log from './util/logging.js';
-import _ from './util/localization.js';
```

Comment:
We should probably move `localization.js` to the app side of things now.

Reply:
Yeah, one small thing to note is that tests/test.util.js is generally written for core/util and will now test app/localization.js as well, but not a big problem

Reply:
Or just split that test?

Reply:
yeah, sounds good